crates/router/tests/connectors/paypal.rs

```diff
@@ -428,27 +428,6 @@ async fn should_sync_refund() {
 }
 
 // Cards Negative scenerios
-// Creates a payment with incorrect card number.
-#[actix_web::test]
-async fn should_fail_payment_for_incorrect_card_number() {
```

Comment:
Why was this test removed? Would using a card number of valid length (16 digits) help?

Reply:
should_fail_payment_for_incorrect_card_number is removed for this connectors mentioned above . As we have luhn check in place, we don't need this test . this test will only validate the invalid card number scenario and not unsupported card scenarios as connectors don't throw INVALID_CARD_NUMBER type error. This is discussed with @jagan-jaya @ArjunKarthik